db/institutions.yml

```diff
@@ -681,6 +681,18 @@ natautoumexico:
   state: ''
   street: Av Universidad 3000, Cd. Universitaria, Coyoacán
   zip: '04510'
+nelsonmandelaafricaninstitutionofscienceandtechnology:
```

Comment:
let's make this nmaist (lower-case) (I know that the program made this automatically, but I don't like it so much)